kvmagent/kvmagent/plugins/vm_metadata/lv_manager.py

```
@@ -0,0 +1,293 @@
+                'vm_uuid': vm_uuid,
+                'lv_path': lv_path,
+                'lv_size': lv_size,
+            })
+    return result
+
+
+# ===================================================================
+# Health Check  (§8.6)
+# ===================================================================
+
+def get_metadata_status(lv_path, lv_size):
```

Ruff (0.15.2)

[warning] Unused function argument: `lv_size`

(ARG001)